lib/common/validation.js

```diff
@@ -717,6 +719,107 @@ function createValidateUUIDFn(field, required) {
     };
 }
 
+function createValidateSortFn(field, options) {
+    assert.string(field, 'field');
+    options = options || {};
+    assert.object(options, 'options');
+
+    return function (params) {
+        var errs = [];
+        var sortString = params[field];
+
+        if (options.required && sortString === undefined)
+            errs.push(errors.missingParamErr(field));
+
+        if (sortString !== undefined && !validateSortParam(sortString)) {
+            errs.push(errors.invalidParamErr(field,
+                'Invalid sort param: ' + sortString));
+        }
+
+        return errs;
+    };
+}
+
+var VALID_SORT_KEYS = [
+    'uuid',
+    'owner_uuid',
+    'image_uuid',
+    'billing_id',
+    'server_uuid',
+    'package_name',
+    'package_version',
+    'tags',
+    'brand',
+    'state',
+    'alias',
+    'max_physical_memory',
+    'create_timestamp',
+    'docker'
+];
+
```

Comment:
No, it just seemed consistent with other parts of the file.  Feel free to ignore!